Cm/pse 08 (#29)
Ada-Developers-Academy/core-pses: 08/kth-missing-positive-number.checkpoint.md

##### !end-tests

### !end-challenge

### !challenge

* type: short-answer
* id: 70178554-c412-45bb-80c9-1b10de5f75cb

> * title: What is your time complexity?

Reviewer: Change to match previous PSEs
```suggestion
* title: What is the time complexity of your solution?
```

> * points: 0

Reviewer: Change to match previous PSEs

```suggestion
* points: 1
```

* topics: Big-O

##### !question

> What is the time complexity (Big O) of your solution?

Reviewer: ```suggestion
What is the time complexity of your solution? Explain. Define your variable(s).
```

##### !end-question

##### !placeholder

> Time Complexity

Reviewer: Change to match previous PSEs
```suggestion
Time Complexity?
```

##### !end-placeholder

##### !answer

/.+/

##### !end-answer

### !end-challenge

### !challenge

* type: short-answer
* id: 143195b2-0991-4c74-9b81-88c2438e9f5f

> * title: What is your space complexity

Reviewer: Change to match previous PSEs

```suggestion
* title: What is the space complexity of your solution?
```

> * points: 0

Reviewer: Change to match previous PSEs
```suggestion
* points: 1
```

* topics: Big-O

##### !question

> What is the space complexity (Big O) of your solution?

Reviewer: ```suggestion
What is the space complexity of your solution?  Explain. Define your variable(s).
```
